audit/audit-gcp.sh

```diff
@@ -36,6 +36,7 @@ gcloud \
     --filter="parent.id=${CNCF_GCP_ORG}" \
     --format="value(name, projectNumber)" \
 | sort \
+| tail -1 | head -1 \
```

Comment:
This is the temporary limit to a single org. I'll remove this once we have all orgs updated.

Reply:
can you move the script update to a separate commit (and/or PR)